src/cf_gen.cc

```diff
@@ -649,7 +649,7 @@ gen_parse(const EntryList &head, std::ostream &fout)
          "parse_line(char *buff)\n"
          "{\n"
          "\tchar\t*token;\n"
-         "\tif ((token = strtok(buff, w_space)) == NULL) \n"
+         "\tif ((token = strtok(buff, \" \\t\")) == NULL) \n"
```

Comment:
This change effectively removes two characters -- ASCII CR and LF -- from strtok() delimiter set, but that removal does _not_ affect the generated parse_line() functionality because parse_line() callers already remove CR and LF from parse_line() input.

This removal was done to make the definition of "space" in IsSpace() and parse_line() the same: Just SP and HT characters. This equality simplifies code (and will allow us to treat both places the same when refactoring parsing code without lengthy explanations or wasteful debates).

P.S. This comment does not request any PR changes.

Reply:
How about defining an additional constant alongside w_space?

Reply:
This is the only place where this constant is used in primary Squid code today, and (after this PR lands) it will be replaced with CharacterSet::WSP in #1840. @kinkie, if you do not mind, to preserve Amos' approval, let's land this PR without code changes. If the constant survives this PR series, I will post a dedicated PR naming it.

Reply:
It's a nit, so sure